README.md

```diff
@@ -43,36 +44,37 @@ This removes all docker images, networks, and local volumes.
     make run
     ```
 
-    This starts all of the services except for the endpointmanager:
+    This starts all of the following services:
     * **Lantern Front End** - The front end for the Lantern application (localhost:8090)
```

Comment:
I might be misinterpreting this, but didn't you remove lantern-front-end from the docker compose files, and therefore this service will not start when running make run?

Reply:
yup! I forgot to remove this from the docs -- nice catch